microservice/microserviceTests/CronBuilderTests/JobConversionTests.cs

```diff
@@ -0,0 +1,38 @@
+using Beamable.Api.Autogenerated.Models;
+using Beamable.Common.Content;
+using Beamable.Common.Scheduler;
+using NUnit.Framework;
+
+namespace microserviceTests.CronBuilderTests;
+
+public class JobConversionTests : CommonTest
+{
+	[TestCase]
+	public void CanDeserializeJobWithoutSource()
+	{
+		var job = new JobDefinition
+		{
+			id = "test",
+			name = "test",
+			owner = "test",
+			source = new OptionalString(),

```

Comment:
The semantics here are that `new OptionalString()` with no constructor argument is an empty optional string? That is, `new OptionalString()` is the equivalent of Scala`s `None`?

Reply:
correct- that is a `None`